typescript/packages/subsurface-viewer/src/storybook/examples/MultiViewExamples.stories.tsx

```diff
@@ -80,6 +83,308 @@ export const MultiViewAnnotation: StoryObj<typeof SubsurfaceViewer> = {
+                        <div>{el.value.toFixed(3)}</div>
+                    </React.Fragment>
+                )
+            ) ?? ""}
+        </div>
+    );
+}
+
+function MultiViewPickingExample(
+    props: SubsurfaceViewerProps
+): React.ReactNode {
+    const [pickingInfoPerView, setPickingInfoPerView] =
+        React.useState<PickingInfoPerView>(
+            props.views?.viewports.reduce((acc, viewport) => {
+                acc[viewport.id] = {
+                    x: null,
+                    y: null,
+                    properties: [],
+                };
+                return acc;
+            }, {} as PickingInfoPerView) ?? {}
+        );
+
+    const deckGlRef = React.useRef<DeckGLRef>(null);
+    const assembler = React.useRef<MultiViewPickingInfoAssembler | null>(null);
+
+    React.useEffect(function onMountEffect() {
+        assembler.current = new MultiViewPickingInfoAssembler(
+            deckGlRef.current
+        );
+
+        const unsubscribe = assembler.current.subscribe((info) => {
+            setPickingInfoPerView(info);
+        });
+
+        return function onUnmountEffect() {
+            unsubscribe();
+        };
+    }, []);
+
+    function handleMouseEvent(event: MapMouseEvent) {
+        if (event.type === "hover") {
+            assembler.current?.getMultiViewPickingInfo(event);
+        }
+    }
+
+    return (
+        <div style={{ width: "100%", height: "90vh", position: "relative" }}>
+            <SubsurfaceViewer
+                {...props}
+                deckGlRef={deckGlRef}
+                onMouseEvent={handleMouseEvent}
+                coords={{
+                    visible: false,
+                    multiPicking: true,
+                }}
+            >
+                {
+                    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                    /* @ts-expect-error */
+                    <View id="view_1">
+                        <ContinuousLegend min={-3071} max={41048} />
+                        <ViewFooter>kH netmap</ViewFooter>
+                        <ExampleReadoutComponent
+                            viewId="view_1"
+                            pickingInfoPerView={pickingInfoPerView}
+                        />
+                    </View>
+                }
+                {
+                    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                    /* @ts-expect-error */
+                    <View id="view_2">
+                        <ContinuousLegend min={2725} max={3396} />
+                        <ViewFooter>Hugin</ViewFooter>
+                        <ExampleReadoutComponent
+                            viewId="view_2"
+                            pickingInfoPerView={pickingInfoPerView}
+                        />
+                    </View>
+                }
+            </SubsurfaceViewer>
+        </div>
+    );
+}
+
+export const MultiViewPicking: StoryObj<typeof SubsurfaceViewer> = {
```

Comment:
Consider adding a `play` function that moves the mouse cursor on top of one of the maps. This should give a more useful story snapshot for catching regressions.

See an example `play` function here:
https://github.com/equinor/webviz-subsurface-components/blob/d7ae8dbecf9e4134aa55fef4fe35030258ed9491/typescript/packages/subsurface-viewer/src/storybook/examples/CameraControlExamples.stories.tsx#L722

Reply:
I implemented a `play` function that moves the cursor to the left-top corner and then to the center of the first viewport.